tox.ini

```diff
@@ -1,10 +1,33 @@
 [tox]
-envlist = py{27,35}
+envlist = clean, py27, py36, report, flake8
 
 [testenv]
-basepython =
-    py27: python2.7
-    py35: python3.5
+deps =
+    -r{toxinidir}/requirements.txt
+    -r{toxinidir}/test-requirements.txt
+    pytest-cov
 sitepackages = False
 commands =
-    python setup.py test
+    pytest --cov --cov-append {posargs}
+
+[testenv:flake8]
+skip_install = true
+deps = flake8
+commands =
+    flake8 reviewrot/
+    flake8 test/
+
+[flake8]
+max-line-length=88
```

Comment:
I'd try to stick to the default line length of 80. You can keep this for now however.